Libplanet/Action/AccountBalanceGetter.cs

```diff
@@ -0,0 +1,19 @@
+#nullable enable
```

Comment:
Is this statement needed?

Reply:
We're going to eventually configure `<Nullable>enable</Nullable>` in project-level, so it's a transition step.

Reply:
I see. then, is it okay we skip `#nullable restore` at the end of the file?

Reply:
Yeah, see also @earlbread's early review on the other PR: <https://github.com/planetarium/libplanet/pull/831#discussion_r396854992>.